doc/classes/VisualServer.xml

```diff
@@ -1247,6 +1280,7 @@
 			<argument index="12" name="bilateral_sharpness" type="float">
 			</argument>
 			<description>
+				Sets the variables to be used with the "screen space ambient occlusion" post-process effect. See [Environment] for more details.
```

Comment:
Same as above, I think "with the Screen Space Ambient Occlusion (SSAO) post-process effect" would read better.